lib/generators/rspec/install/install_generator.rb

```diff
@@ -41,6 +43,10 @@ def generate_rspec_init(tmpdir)
 
         replace_generator_command(spec_helper_path)
         remove_warnings_configuration(spec_helper_path)
+
+        dot_rspec_path = File.join(tmpdir, '.rspec')
+
+        append_default_path(dot_rspec_path)
```

Comment:
Its not necessary to always set this, perhaps:
```suggestion
        unless default_path == "spec"
          dot_rspec_path = File.join(tmpdir, '.rspec')

          append_default_path(dot_rspec_path)
        end
```

Although it would be nice if we could determine if it was set when generating.